experiments/configs/hugging-face/eval_configs/eval_pipeline_config.yaml

```diff
@@ -0,0 +1,8 @@
+model: hf-causal
+model_args: "pretrained=/fsx/proj-chemnlp/experiments/checkpoints/finetuned/full_1b/checkpoint-final"
```

Comment:
also, unrelated to this PR but at some point we should add docs similar to [this](https://github.com/OpenBioML/chemnlp/blob/main/docs/EXPERIMENT.md) and [this](https://github.com/OpenBioML/chemnlp/tree/main/experiments) for how to use the scripts.